schema/deploy/shipping/views.sql

```diff
@@ -49,6 +49,10 @@ drop materialized view if exists shipping.fhir_questionnaire_responses_v1;
 
 drop view if exists shipping.sample_with_best_available_encounter_data_v1;
 
+drop view if exists shipping.uw_reopening_ehs_reporting_v1;
+drop view if exists shipping.uw_reopening_encounters_v1;
+drop view if exists shipping.uw_reopening_enrollment_fhir_encounter_details_v1;
```

Comment:
Moved. Boy oh boy. Why not drop view if exists blah cascade ? Then we wouldn't have to worry about the exact order.

Reply:
Hmm that's a good question. I personally think it's better to be explicit about the drop statements since they are destructive, but we can definitely have a discussion about this.